docs/pages/workflow/debugging.md

```diff
@@ -55,41 +56,65 @@ This might indicate that there is a performance issue. You likely need to run yo
 
 The Expo community and the React and React Native communities are great resources for help when you get stuck. There's a good chance someone else has run into the exact same error as you, so make sure to read the documentation, search the [forums](https://forums.expo.dev/), [GitHub issues](https://github.com/expo/expo/issues/), and [StackOverflow](https://stackoverflow.com/).
 
-## Useful tools for debugging
+## Debugging Tools
 
 Below are a few tools we recommend, and use ourselves, when it comes to debugging your Expo app:
 
-## Developer menu
+### Developer Menu
```

Comment:
```suggestion
### Developer menu
```

We can use sentence case here. It's a general menu, not something specific to the Expo Go app. Keeping headers to follow sentence casing is also mentioned in the [Expo's writing style guide](https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md#headings).